packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConsoleLogger } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
+import { Logger } from '@opentelemetry/types';
+import { Meter } from '.';
+import { DEFAULT_CONFIG, MeterConfig } from './types';
+
+/**
+ * This class represents a meter provider which platform libraries can extend
+ */
+export class MeterProvider implements types.MeterProvider {
+  private readonly _meters: Map<string, Meter> = new Map();
+  readonly logger: Logger;
+
+  constructor(private _config: MeterConfig = DEFAULT_CONFIG) {
+    this.logger = _config.logger || new ConsoleLogger(_config.logLevel);
+  }
+
+  getMeter(
+    name: string = 'default',
```

Comment:
@mayurkale22 Just to double check that we decide to have 'default' name and also give better names in examples and tests in today's SIG metting, is it correct? Thanks :)

Reply:
No, this typing has `name` as an optional property. We want it to be required, but we need to check anyways since someone not using typescript may call `getMeter` without name and we have to return something